pixi.toml

```diff
@@ -11,22 +11,24 @@ build_sdist = "pixi run python -m build --sdist"
 
 [dependencies]
 python = ">=3.10"
-build = "*"
-rattler-build = "*"
-conda-build = "*"
-"ruamel.yaml" = "*"
+build = ">=0.7.0,<0.8"
+rattler-build = ">=0.18.1,<0.19"
```

Comment:
This is for sure going to cause issues. We release minor bumps on a regular basis. Perhaps using `1.0` here makes sense ... ?